<launch>
    <node pkg="speech_to_text" type="speech_to_text_node.py" name="speech_to_text_node" output="screen">
        <param name="sampling_frequency" value="16000"/>
        <param name="frame_sample_count" value="92"/>
        <param name="request_frame_count" value="20"/>
        <param name="language" value="fr"/>

        <remap from="audio_in" to="audio_signed_16_16000"/>
        <remap from="audio_in/filter_state" to="speech_to_text/filter_state"/>
        <remap from="text" to="speech_to_text/transcript"/>
    </node>
</launch>
